scripts/combine_control_files.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import sys
+from os.path import isdir, join
+
+# import shutil
+
+pg_version = sys.argv[1]
+#  build_tag = sys.argv[2]
```

Comment:
Why don't we use build_tag?
Should it be uncommented?

Reply:
we only need it once we do de-duplication #4789

Reply:
got it.
please, add a comment about that

Reply:
I'm just deleting the code for now, put it in #4789